cmd/snapctl/config.go

```diff
@@ -77,24 +77,24 @@ func getConfig(ctx *cli.Context) {
 	if ptyp == "" {
 		fmt.Println("Must provide plugin type")
 		cli.ShowCommandHelp(ctx, ctx.Command.Name)
-		os.Exit(1)
+		return errCritical
```

Comment:
Won't `cli` handle printing the error text? It may be better to move the `fmt.Println` text into a custom error message here instead of returning a generic one.